go/exec/status.go

```diff
@@ -18,9 +18,46 @@
 
 package exec
 
+import (
+	"github.com/vitessio/arewefastyet/go/storage"
+)
+
 const (
 	StatusCreated  = "created"
 	StatusStarted  = "started"
 	StatusFailed   = "failed"
 	StatusFinished = "finished"
 )
+
+type benchmarkStats struct {
+	Benchmark_total    int
+	Benchmark_finished int
+	Benchmark_30Days   int
+}
+
+func GetBenchmarkStats(client storage.SQLClient) (benchmarkStats, error) {
+	rows, err := client.Select(`
+        SELECT
+            (SELECT COUNT(uuid) FROM execution) AS count_status,
+            (SELECT COUNT(*) FROM execution WHERE status = 'finished') AS count_finished,
+            (SELECT COUNT(*) FROM execution WHERE started_at >= DATE_SUB(CURDATE(), INTERVAL 30 DAY)) AS count_all
+        FROM
+            execution
+		LIMIT 1;
```

Comment:
I don't think the `LIMIT 1` is required here as `COUNT`s always return one row anyway